apps/web-giddh/src/app/tb-pl-bs/tb-pl-bs.component.ts

```diff
@@ -41,15 +44,46 @@ export class TbPlBsComponent implements OnInit, OnDestroy {
     /* This will hold common JSON data */
     public commonLocaleData: any = {};
 
-    constructor(private store: Store<AppState>, private companyActions: CompanyActions, private _route: ActivatedRoute, private router: Router, private breakPointObservar: BreakpointObserver,) {
+    constructor(private store: Store<AppState>, private companyActions: CompanyActions, private _route: ActivatedRoute, private router: Router, private _generalActions: GeneralActions, private breakPointObservar: BreakpointObserver,) {
```

Comment:
Avoid introduction of unused dependencies `_generalActions`

Reply:
Done